python/cuml/test/dask/test_random_forest.py

```diff
@@ -51,14 +57,18 @@ def test_rf_classification():
 
     X_cudf = cudf.DataFrame.from_pandas(pd.DataFrame(X_train))
     X_train_df = \
-        dask_cudf.from_cudf(X_cudf, npartitions=len(workers)).persist()
+            dask_cudf.from_cudf(X_cudf, npartitions=len(workers))
```

Comment:
I think we should test both with the `.persist()` call explicitly as well as using the (shiny and awesome) `dask_utils.persist_across_workers` since right now this are early days so I could see users might prefer one way over the other (perhaps they had a prior .persist() call in their code or something)

Reply:
I can add an example of doing persist the proper way, which would look like:

* `X_train_df, y_train_df = c.persist([X_train_df, y_train_df], workers={X_train_df=workers, y_train_df=workers})`

If you just use `foo.persist()` as in this old line, you're gonna have a bad time...

Reply:
Actually, I think I'd prefer to add this in a doc rather than a test, since it will be completely equivalent to the existing code...

Reply:
Hmm.. I added the doc but it looks rather bad... maybe it's not what you had in mind?